src/hats_import/catalog/arguments.py

```diff
@@ -128,40 +128,23 @@ def _check_arguments(self):
         # Basic checks complete - make more checks and create directories where necessary
         self.input_paths = find_input_paths(self.input_path, "**/*.*", self.input_file_list)
 
-    def to_table_properties(self, total_rows: int) -> TableProperties:
+    def to_table_properties(
+        self, total_rows: int, highest_order: int, moc_sky_fraction: float
+    ) -> TableProperties:
         """Catalog-type-specific dataset info."""
         info = {
             "catalog_name": self.output_artifact_name,
             "catalog_type": self.catalog_type,
             "total_rows": total_rows,
             "ra_column": self.ra_column,
             "dec_column": self.dec_column,
-        }
+            "hats_cols_sort": self.sort_columns,
+            "hats_max_rows": self.pixel_threshold,
+            "hats_order": highest_order,
+            "moc_sky_fraction": f"{moc_sky_fraction:0.5f}",
+        } | self.extra_property_dict()
```

Comment:
I did not know until just now that `|` combined dictionaries! 🎓

Reply:
I learned yesterday =D